src/schemas/ev42/ev42_rw.cpp

```diff
@@ -117,26 +112,65 @@ void HDFWriterModule::parse_config(std::string const &ConfigurationStream,
     Logger->trace("buffer_packet_max: {}", buffer_packet_max);
   } catch (...) { /* it's ok if not found */
   }
+  try {
+    RecordAdcPulseDebugData =
+        ConfigurationStreamJson["adc_pulse_debug"].get<bool>();
+    Logger->trace("adc_pulse_debug: {}", RecordAdcPulseDebugData);
+  } catch (...) { /* it's ok if not found */
+  }
+}
+
+void HDFWriterModule::createAdcGroupAndDatasets(hdf5::node::Group &HDFGroup) {
+  auto AdcGroup = HDFGroup.create_group(AdcGroupName);
```

Comment:
When discussing the layout of the debug data with Tobias, he suggested that it be put next to the other datasets (i.e. in the `HDFGroup`-group). I do not have a strong opinion on the matter however so I suggest that we leave @zjttoefs until Friday afternoon to comment. We can easily change things later.

Reply:
Yes, I wasn't sure which to go for, in the end opted for the group because I would otherwise have to prefix the datasets with what I used as the group name. On the other hand being in the same group might be cleared that they should be consistent with `event_time_zero` and `event_index`.

Reply:
As it was Tobias's suggestion I've removed the group. The ADC datasets now have a prefix of `adc_pulse_` (`adc_pulse_amplitude`, etc) and are in the `NXevent_data` group.